package.json

```diff
@@ -11,10 +11,13 @@
     "prepare": "husky install"
   },
   "dependencies": {
-    "autoprefixer": "^10.4.19",
-    "postcss": "^8.4.38",
+    "@tanstack/react-query": "^5.49.2",
+    "axios": "^1.7.2",
+    "classnames": "^2.5.1",
     "react": "^18.3.1",
-    "react-dom": "^18.3.1"
+    "react-dom": "^18.3.1",
+    "react-query": "^3.39.3",
```

Comment:
i think it's not need, need use from tanstack

Reply:
react-dom needed